src/llmq/context.cpp

```diff
@@ -45,7 +45,7 @@ LLMQContext::LLMQContext(CChainState& chainstate, CConnman& connman, CDeterminis
         return llmq::quorumInstantSendManager.get();
     }()},
     ehfSignalsHandler{std::make_unique<llmq::CEHFSignalsHandler>(chainstate, mnhfman, *sigman, *shareman, mempool,
-                                                                 *qman, sporkman, peerman)}
+                                                                 *qman, peerman)}
```

Comment:
clang format complains:
```suggestion
    ehfSignalsHandler{
        std::make_unique<llmq::CEHFSignalsHandler>(chainstate, mnhfman, *sigman, *shareman, mempool, *qman, peerman)}
```